projects/arlas-toolkit/src/lib/components/timeline/timeline/timeline.component.ts

```diff
@@ -123,7 +138,7 @@ export class TimelineComponent implements OnInit {
         });
       });
       if (this.detailedTimelineComponent) {
-        this.detailedTimelineComponent.input.chartHeight = 75;
+        this.detailedTimelineComponent.input.chartHeight = 76;
```

Comment:
Why adding `1` pixel ?

Reply:
Without this pixel, the timeline + detailed timeline was 129px high instead of 130px like it is when there is no detailed timeline